apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1811,7 +1816,7 @@ create_tx_for_signing(#data{opts = #{ initiator        := Initiator
          , lock_period      => LockPeriod },
     %% nonce is not exposed to the client via WebSocket but is used in tests
     %% shall we expose it to the client as well?
-    Optional = maps:with([nonce], Opts),
+    Optional = maps:with([nonce, fee], Opts),
```

Comment:
For consistency, perhaps this could have been changed to `XOpts` as well? :)

Reply:
Generally this should be cleaned up across the SC code base. This originated from the occasion where there were option, and some extra options I believe. The naming is confusing at least.

Reply:
@uwiger `XOpts` are the accumulation of optional and mandatorary maps of properties. This is only the optional one. I do agree with @tolbrino that this is a bit inconsistent :D